emission/tests/netTests/TestHabiticaRegister.py

```diff
@@ -0,0 +1,39 @@
+# Standard imports
+import unittest
+import logging
+import json
+import uuid
+import attrdict as ad
+
+# Our imports
+import emission.net.ext_service.habitica.register as reg
+import emission.core.get_database as edb
+
+class TestHabiticaRegister(unittest.TestCase):
+  def setUp(self):
+    print "Test setup called"
+    self.testUserUUID = uuid.uuid4()
+    self.sampleAuthMessage1 = {'username': '99999999999995543', 'email': '99999999999995543@test.com', 'password': 'aa!A15551', 'our_uuid': self.testUserUUID}
+
+  # One part of this is the communication with habitica. That is an integration test, not a unit test,
```

Comment:
This is not accurate - this does actually test the communication with habitica since the register code makes a call to the server.